Makefile

```diff
@@ -41,6 +43,9 @@ $(TSC):
 bower_components:
 	bower install
 
+lint:
+	$(TSLINT) $(foreach source,$(TSSOURCES), -f $(source))
```

Comment:
`./node_modules/tslint/bin/lslint -f src/integer.ts src/interpreter.ts`
gives output only for `src/integer.ts` for me.